server/go/pkg/server/database.go

```diff
@@ -6,6 +6,27 @@ import (
 	"github.com/go-sql-driver/mysql"
 )
 
+var (
+	dbconnection *sql.DB
+)
+
+func getMysql(connStr string, pool bool) (*sql.DB, error) {
```

Comment:
can merge this function into `newMysql` if we eliminate the new option as recommended.